<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Planning</class>
 <widget class="QDialog" name="Planning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>51</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="planning">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Planning</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="planningWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Category</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descriptions</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>321</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="category_2">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="money_2">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Money</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="description_2">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="CategoryBox">
            <item>
             <property name="text">
              <string>Food</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Education</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Transportation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Clothes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Entertainment</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Communication fee</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>If you want add more input name and press +</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLineEdit" name="line_money_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>$</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLineEdit" name="line_descript_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="year">
      <property name="text">
       <string>Year </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Text_year"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="month">
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_month">
      <item>
       <property name="text">
        <string>Jan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Feb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jun</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aug</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sep</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oct</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nov</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dec</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton_2">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancle_Button_3">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
